src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -47,6 +52,8 @@ MXNET_OPERATOR_REGISTER_UNARY_LAUNCH_DR(sigmoid, cpu, kernel_launch_op::sigmoid)
 .. math::
    y = 1 / (1 + exp(-x))
 
+The storage type of ``sigmoid`` output is always denseThe storage type of ``sigmoid`` output is always dense
```

Comment:
Do people know what 'default' is as easily as they will know what 'dense' is (for example, someone who is not familiar with sparse tensors)?  I don't have it quoted.  I can change it, but I feel this is more clear.